### PR TITLE
[SMALLFIX]Update static import

### PR DESCRIPTION
**Pull request by XiaoMuxin**

### CODE REVIEW

File: core/common/src/test/java/alluxio/util/io/ByteIOUtilsTest.java [hunk 1]
```diff
@@ -29,7 +30,8 @@ public void readWriteByte() {
     for (long i : values) {
       byte v = (byte) i;
       ByteIOUtils.writeByte(mBuf, 0, v);
-      Assert.assertEquals(v, ByteIOUtils.readByte(mBuf, 0));
+      //Assert.assertEquals(v, ByteIOUtils.readByte(mBuf, 0));
```
Comment (apc999) [inline core/common/src/test/java/alluxio/util/io/ByteIOUtilsTest.java]: remove

File: core/common/src/test/java/alluxio/util/io/ByteIOUtilsTest.java [hunk 2]
```diff
@@ -42,7 +44,8 @@ public void readWriteShort() {
     for (long i : values) {
       short v = (short) i;
       ByteIOUtils.writeShort(mBuf, 0, v);
-      Assert.assertEquals(v, ByteIOUtils.readShort(mBuf, 0));
+      // Assert.assertEquals(v, ByteIOUtils.readShort(mBuf, 0));
```
Comment (apc999) [inline core/common/src/test/java/alluxio/util/io/ByteIOUtilsTest.java]: remove

File: core/common/src/test/java/alluxio/util/io/ByteIOUtilsTest.java [hunk 3]
```diff
@@ -58,7 +61,8 @@ public void readWriteInt() {
       int v = (int) i;
       for (int pos = 0; pos < 4; pos++) {
         ByteIOUtils.writeInt(mBuf, pos, v);
-        Assert.assertEquals(v, ByteIOUtils.readInt(mBuf, pos));
+        assertEquals(v, ByteIOUtils.readInt(mBuf, pos));
+	//Assert.assertEquals(v, ByteIOUtils.readInt(mBuf, pos));
```
Comment (apc999) [inline core/common/src/test/java/alluxio/util/io/ByteIOUtilsTest.java]: remove

File: core/common/src/test/java/alluxio/util/io/ByteIOUtilsTest.java [hunk 4]
```diff
@@ -74,7 +78,8 @@ public void readWriteLong() {
     for (long v : values) {
       for (int pos = 0; pos < 8; pos++) {
         ByteIOUtils.writeLong(mBuf, 0, v);
-        Assert.assertEquals(v, ByteIOUtils.readLong(mBuf, 0));
+        //Assert.assertEquals(v, ByteIOUtils.readLong(mBuf, 0));
```
Comment (apc999) [inline core/common/src/test/java/alluxio/util/io/ByteIOUtilsTest.java]: remove

**Comment by newnius** [comment]
@apc999 Duplicate https://github.com/Alluxio/alluxio/pull/9080

**Comment by apc999** [comment]
Thanks. close this PR as duplicate of #9080

**Pull request closed without merging**